source/Cosmos.HAL2/PS2Mouse.cs

```diff
@@ -137,19 +144,18 @@ public void HandleMouse(ref INTs.IRQContext context)
             else if (mMouseCycle == 2)
             {
                 mMouseByte[2] = IO.Data.Byte;
-
-                if (HasScrollWheel)
```

Comment:
Why remove the check for a ScrollWheel?
Not every PS/2 mouse has one, so please put the checks back in place